### PR TITLE
Image Viewer Continued

### PR DESCRIPTION
**Pull request by linuxgurugamer**

### CODE REVIEW

File: NetKAN/ImageViewerCont.netkan [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "spec_version": "v1.4",
+  "install": [
+    {
+      "install_to": "GameData",
+      "find": "ImageViewer"
+    }
+  ],
+  "$vref": "#/ckan/ksp-avc",
```
Comment (politas) [inline NetKAN/ImageViewerCont.netkan]: "$vref": ""#/ckan/ksp-avc/GameData/ImageViewer/ImageViewerCont.version",

Comment (linuxgurugamer) [inline NetKAN/ImageViewerCont.netkan]: Done, but why was it needed?

Comment (Olympic1) [inline NetKAN/ImageViewerCont.netkan]: If there are multiple version files in a download, ckan doesn't know which one to pick and fails. Adding a path to the vref tells which file to use.

**Comment by politas** [comment]
Merged with my thanks!

**Pull request merged**